tests/test_pages.py

```diff
@@ -430,3 +436,151 @@ def nextPage(self):
         assert request.metadata is not None
         assert request.metadata.probability == 1.0
     assert type(page.nextPage) == Request
+
+
+def test_price_selector():
```

Comment:
What do you think about moving all these price tests to a separate file? test_pages becomes large.